tests/sources/test_rng_manual_input_source.py

```diff
@@ -0,0 +1,47 @@
+from unittest import mock
+import sys
+import pytest
+
+from telliot_feeds.sources.manual_sources.tellor_rng_manual_source import TellorRNGManualInputSource
+
+
+@pytest.mark.asyncio
+async def test_rng_valid_input(capsys):
+    with mock.patch("builtins.input", side_effect=["0x2b563420722cbcfc84857129bef775e0dc5f1401", ""]):
+        result, _ = await TellorRNGManualInputSource().fetch_new_datapoint()
+        expected = "TellorRNG value to be submitted on chain: 2b563420722cbcfc84857129bef775e0dc5f1401\nPress [ENTER] to continue"
+        captured_output = capsys.readouterr()
+        assert expected in captured_output.out.strip()
+        assert result == bytes.fromhex("2b563420722cbcfc84857129bef775e0dc5f1401")
+
+
+@pytest.mark.asyncio
+async def test_rng_invalid_input(capsys):
+    with mock.patch("builtins.input", side_effect=["exit"]):
+        try:
+            result, _ = await TellorRNGManualInputSource().fetch_new_datapoint()
+        except RuntimeError as e:
```

Comment:
For these, use `with pytest.raises(...` instead. link: https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions